engine/release-notes/index.md

```diff
@@ -25,6 +25,10 @@ for Docker Engine.
 ## 20.10.8
 2021-08-03
 
+**IMPORTANT**: Due to [net/http changes](https://github.com/golang/go/issues/40909) in [Go 1.16](https://golang.org/doc/go1.16#net/http),
+setting an `$HTTP_PROXY` environment variable alone is not enough any more to properly configure an HTTP proxy.
+Make sure you also set an `$HTTPS_PROXY` environment variable, for handling requests to `https://` URLs.
+
```

Comment:
Thanks! I had a quick go at slightly reformatting the block to be rendered as a "note" (to make it stand out better), and adding a link the the relevant section in the docs on configuring a proxy.

```suggestion
> **IMPORTANT**
>
> Due to [net/http changes](https://github.com/golang/go/issues/40909) in [Go 1.16](https://golang.org/doc/go1.16#net/http),
> HTTP proxies configured through the `$HTTP_PROXY` environment variable are no
> longer used for TLS (`https://`) connections. Make sure you also set an `$HTTPS_PROXY`
> environment variable for handling requests to `https://` URLs.
>
> Refer to the [HTTP/HTTPS proxy section](../../config/daemon/systemd.md#httphttps-proxy)
> to learn how to configure the Docker Daemon to use a proxy server.
```

With the above suggestion applied, it looks like:

<img width="999" alt="Screenshot 2021-08-04 at 13 54 22" src="https://user-images.githubusercontent.com/1804568/128176618-1db4fe20-161a-4866-a87b-623a24180ce4.png">

<img width="985" alt="Screenshot 2021-08-04 at 13 54 36" src="https://user-images.githubusercontent.com/1804568/128176625-3af7d897-e707-4d04-9a9f-7299ca6c4918.png">

Reply:
Thanks @thaJeztah. Should we also use `{: .important }` as we have an admonition defined for Important notes. Also, can we change the uppercase IMPORTANT to Important?

Reply:
Gave it a quick go, and works for me! I updated the suggestion

<img width="992" alt="Screenshot 2021-08-04 at 14 05 54" src="https://user-images.githubusercontent.com/1804568/128177922-dc020aca-2352-4434-991a-131bbd7a69c2.png">
<img width="985" alt="Screenshot 2021-08-04 at 14 06 01" src="https://user-images.githubusercontent.com/1804568/128177923-6bf67d18-9058-48c7-910b-32d15244474e.png">